contentrepo/settings/production.py

```diff
@@ -18,6 +18,7 @@
 CSRF_TRUSTED_ORIGINS = env.list("CSRF_TRUSTED_ORIGINS")
 
 SENTRY_DSN = env.str("SENTRY_DSN", "")
+SENTRY_ENVIRONMENT = env.str("SENTRY_ENVIRONMENT", "")
```

Comment:
The default environment is `production`, all lower case.

I think having it be an empty string is invalid, which makes it really easy to have an invalid configuration if you specify `SENTRY_DSN` but not `SENTRY_ENVIRONMENT`.

So I think it will be better to default this to `production` instead of an empty string.

Reply:
I see, I've changed it. It should be okay now.